package.json

```diff
@@ -37,7 +37,7 @@
     "@material-ui/core": "^4.11.3",
     "@material-ui/icons": "^4.11.2",
     "@psychedelic/plug-controller": "0.7.6",
-    "@psychedelic/plug-inpage-provider": "1.4.1",
+    "@psychedelic/plug-inpage-provider": "../plug-inpage-provider",
```

Comment:
👀 When this happens jsut add a [WIP] on the pr title or a ref to the PR that needs to be merged before this 🖖